File: Ananya Chandra/words.java
//number of words in sentence
import java.util.*;
class wordnum
{
public static void main()
{
 Scanner sc=new Scanner(System.in);
System.out.println("Enter a sentence");   
String a=sc.nextLine(); 
System.out.println("The string is -"+a);
int z=a.length(),y,x=0;
for(y=0;y<z;y++)
{
if(a.charAt(y)==' ')
x=x+1;
}
System.out.println("Number of words in string ="+(x+1));
}
}

File: Ananya Chandra/words.py
#to print words of a camelcase string seperate and print the number of these words
st= input("Enter the string \n")
st=st+"A"
t=0
c=0
l= len(st)
for i in range(1,l):
    if ord(st[i])>=65 and ord(st[i])<=90:
        print(st[t:i])
        t=i
        c+=1

print(c)
